src/AddressRegistry.Api.Legacy/Address/Query/AddressBosaQuery.cs

```diff
@@ -79,7 +79,9 @@ public async Task<AddressBosaResponse> Filter(BosaAddressRequest filter)
                 filter?.Straatnaam?.Taal,
                 filter?.Straatnaam?.SearchType,
                 streetNamesQuery,
-                filteredMunicipalities).ToList();
+                filteredMunicipalities)
+                    .Select(x => new { x.StreetNameId, PersistentLocalId = x.PersistentLocalId, x.Version, x.NisCode })
```

Comment:
Is this needed? `PersistentLocalId = x.PersistentLocalId`, I would guess just `x.PersistentLocalId` is enough, since its the same name?